.github/workflows/natlibfi-node-tests.yml

```diff
@@ -0,0 +1,69 @@
+name: NatLibFi node tests
+
+on: push
+
+jobs:
+  build-node-versions:
+    name: Node version matrix
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x, 18.x, 19.x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+    steps:
+    - name: Checkout the code
+      uses: actions/checkout@v3
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+      env:
+        NPM_CONFIG_IGNORE_SCRIPTS: true
+    - run: npm update
+    - run: npm audit --package-lock-only --production --audit-level=high
+    - run: npm ci
```

Comment:
16.x and 18.x change changed lock file that might throw some error in some point

Reply:
Dropped 16.x so this is resolved.